server/src/Q3Server/Startup.cs

```diff
@@ -32,7 +33,12 @@ public void Configuration(IAppBuilder app)
                     app.CreateLogger<QHub>()));

```

Comment:
I still don't understand why our requests are closs-domain. Also, supposedly using JSONP is less secure...
I've left everything as Rachael has written it.